crypto/signify.go

```diff
@@ -75,14 +75,24 @@ func SignifySignFile(input string, output string, key string) error {
 		return err
 	}
 
+	rawSig := ed25519.Sign(skey, filedata)
 	header := keydata[:2]
 	keyNum := keydata[32:40]
 
 	var sigdata []byte
 	sigdata = append(sigdata, header...)
 	sigdata = append(sigdata, keyNum...)
-	sigdata = append(sigdata, ed25519.Sign(skey, filedata)...)
+	sigdata = append(sigdata, rawSig...)
 
 	out.WriteString(fmt.Sprintf("untrusted comment: verify with geth.pub\n%s\n", base64.StdEncoding.EncodeToString(sigdata)))
+
+	// Add the trusted comment if available (minisign only)
+	if trustedComment != "" {
+		var sigAndComment []byte
+		sigAndComment = append(sigAndComment, rawSig...)
+		sigAndComment = append(sigAndComment, []byte(trustedComment)...)
+		out.WriteString(fmt.Sprintf("trusted comment: %s\n%s\n", trustedComment, base64.StdEncoding.EncodeToString(ed25519.Sign(skey, sigAndComment))))
```

Comment:
Perhaps an error should be generated in case the `trustedComment ` contains linebreaks (particularly `\n` -- I'm not sure what the spec states about other types of whitespace)

Reply:
I still haven't found a spec, but looking at the minisign verification function, it's pretty clear that the comment can only be one line long.

Reply:
just committed the check for this